docs/setup_guides/hpc.rst

```diff
@@ -109,10 +111,114 @@ Copy that string into your ``jupyter_notebook_config.py`` config file
 
-Great. We're done with that.
+For security reasons, we recommend making sure your ``jupyter_notebook_config.py``
+is readable only by you. For more information on and other methods for
+securing Jupyter, check out
+`Securing a notebook server <http://jupyter-notebook.readthedocs.io/en/stable/public_server.html#securing-a-notebook-server>`__
+in the Jupyter documentation.
+
+------------
+
+From here, we have two options. Option 1 will start a Jupyter Notebook Server
+and manage dask and using the `dask-jobqueue`_ package. Option 2 will start a dask
+cluster using `dask-mpi` and will run a Jupyter server as part of the dask cluster.
+
+Deploy Option 1: Jupyter + dask-jobqueue
+----------------------------------------
+
+Start a Jupyter Notebook Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that we have Jupyter configured, we can start a notebook server. In many
+cases, your system administrators will want you to run this notebook server in
+an interactive session on a compute node. This is not universal rule, but it is
+one we'll follow for this tutorial.
+
+In our case, the Cheyenne super computer uses the PBS job scheduler, so typing:
+
+::
+
+    (pangeo) $ qsub -I -l select=1:ncpus=4 -l walltime=03:00:00 -q regular
+
+This will get us an interactive job on the `regular` queue for three hours. You
+may not see the `pangeo` environment anymore in your prompt, in this case, you
+will want to reactivate it.
+
+::
+
+    source activate pangeo
+
+From here, we can start jupyter. The Cheyenne computer administrators have
+developed a `start-notebook <https://www2.cisl.ucar.edu/resources/computational-systems/cheyenne/software/jupyter-and-ipython#notebook>`__
+utility that wraps the following steps into a single execution. You should
+check with your system administrators to see if they have something similar.
+If not, you'll need to take the following steps:
+
+Copy this line into your terminal. It will echo a command you'll want to use
+later.
+
+::
 
-Launch Dask with a script
--------------------------
+    (pangeo) $ echo "ssh -N -L 8877:`hostname`:8877 -L 8878:`hostname`:8787 $USER@cheyenne.ucar.edu"
+    ssh -N -L 8877:r8i4n0:8877 -L 8878:r8i4n0:8787 username@cheyenne.ucar.edu
+
+Now we can launch the notebook server:
+
+::
+
+    (pangeo) $ jupyter lab --no-browser --ip=`hostname` --port=8877
+    [I 13:36:52.298 LabApp] JupyterLab beta preview extension loaded from /home/username/miniconda3/envs/pangeo/lib/python3.6/site-packages/jupyterlab
+    [I 13:36:52.298 LabApp] JupyterLab application directory is /home/username/miniconda3/envs/pangeo/share/jupyter/lab
+    [I 13:36:52.321 LabApp] Serving notebooks from local directory: /home/username
+    [I 13:36:52.321 LabApp] 0 active kernels
+    [I 13:36:52.321 LabApp] The Jupyter Notebook is running at:
+    [I 13:36:52.321 LabApp] http://r8i4n0:8877/
+    [I 13:36:52.321 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
```

Comment:
You might want to avoid the logs here.  They might be a bit scary.

Reply:
I agree. However sometimes they contain useful information for the user to see, especially if something goes wrong (and they are left in the dark, asking support for "it used to work, now it doesn't"). Perhaps we can grep-out only very standard lines and still print the rest, if any?

Reply:
I've shortened the log.